app/controllers/users_controller.rb

```diff
@@ -27,41 +27,64 @@ def find_matching_had_wanted(had_skill_id, wanted_skill_id)
     (find_matching_wanted(had_skill_id) & find_matching_had(wanted_skill_id))
   end
 
-  # Make method return by miles.  Pass a default argument of 5 miles.
+  def find_by_distance(found_users, miles)
+    users = []
+    found_users.each do |user|
+      distance = user.distance_to(current_user)
+      if distance == miles
+        users << user
+      end
+    end
+    users
+  end
+
   def nearby_users
     city = @user.address
     User.near(city)
   end
 
+  def get_nearby_users(miles)
```

Comment:
Unused method argument - `miles`. If it's necessary, use `_` or `_miles` as an argument name to indicate that it won't be used. You can also write as `get_nearby_users(*)` if you want the method to accept any arguments but don't care about them.